exareme2/controller/services/exaflow/tasks_handler.py

```diff
@@ -0,0 +1,31 @@
+from exareme2.controller import logger as ctrl_logger
+from exareme2.controller.celery.tasks_handler import WorkerTasksHandler
+
+
+class ExaFlowTasksHandler:
+    def __init__(
+        self,
+        request_id: str,
+        worker_id: str,
+        worker_queue_addr: str,
+        tasks_timeout: int,
+    ):
+        self._request_id = request_id
+        self._worker_id = worker_id
+        self._worker_queue_addr = worker_queue_addr
+        self._tasks_timeout = tasks_timeout
+        self._logger = ctrl_logger.get_request_logger(request_id=request_id)
+        self._worker_tasks_handler = WorkerTasksHandler(
+            self._worker_queue_addr, self._logger
+        )
+
+    @property
+    def worker_id(self) -> str:
+        return self._worker_id
+
+    def run_step(self, method_name, params: dict) -> dict:
```

Comment:
This needs to be broken to 2 methods, similar to exareme2 so it can be run async.